timeless/views.py

```diff
@@ -8,9 +8,13 @@ class UserView(AdminView):
 @todo #311:30min Once CreateView is implemented, refactor all blueprint views
  to use it for validating the form and storing the record in the database.
  Views already implemented: ItemCreateView
-@todo #173:30min Once UpdateView is implemented, refactor all blueprint views
+@todo #311:30min Once UpdateView is implemented, refactor all blueprint views
  to use it for validating the form and updating the record in the database.
  Reuse SingleObjectMixin to provide simple solution to fetch by id.
+ Views already implemented: ItemCreateView
+@todo #173:30min Once DeleteView is implemented, refactor all blueprint views
```

Comment:
@paulodamaso hmm, this puzzle was already removed, you are trying necromancy? :)

Reply:
@krzyk Well looks like that the ritual didn't worked... :) I'm confused about all these puzzles concerning the views, I've already opened an issue